src/Contributor/ContributorRelationsConfigurator.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Contributor;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\DBAL\Schema\Table;
+use Doctrine\DBAL\Types\Type;
+
+final class ContributorRelationsConfigurator
```

Comment:
Nit: This seems to be only used in unit tests, so maybe move it to the `tests` folder